### PR TITLE
[WIP] add test to check output.txt headers, contents

### PR DESCRIPTION
**Pull request by marisalim**
* adding a test to `tests/test_lca.py` that checks the column header and content of output.txt from the `command_summarize.py` script. until i fix the header and row lines in `command_summarize.py` this should make a test fail in #1540

### CODE REVIEW

File: tests/test_lca.py [hunk 1]
```diff
@@ -1527,6 +1527,32 @@ def test_single_summarize_to_output():
...
+
+def test_single_summarize_to_output_check_filename():
+    with utils.TempDirectory() as location:
+        db1 = utils.get_test_data('lca/delmont-1.lca.json')
+        input_sig = utils.get_test_data('lca/TARA_ASE_MAG_00031.sig')
+        in_dir = os.path.join(location, 'sigs')
+        os.mkdir(in_dir)
+        shutil.copyfile(input_sig, os.path.join(in_dir, 'q.sig'))
+
+        cmd = ['lca', 'summarize', '--db', db1, '--query', os.path.join(in_dir, 'q.sig'),
```
Comment (ctb) [inline tests/test_lca.py:1539]: side note, the use of `in_dir` here should render the use of `in_location` unnecessary in `utils.runscript(...)` - not a big deal, just mentioning it!

Comment (marisalim) [reply tests/test_lca.py:1539]: ah, i see! i kept getting path errors from the lines below so i switched to setting the path with `os.path.join`. how would you write the code below using the `in_location` approach?

```
cmd = ['lca', 'summarize', '--db', db1, '--query', 'q.sig',
               '-o', os.path.join(location, 'output.txt')]
status, out, err = utils.runscript('sourmash', cmd, in_location=location)
```

Comment (ctb) [reply tests/test_lca.py:1539]: I think the disconnect is that `in_dir` is actually f"{location}/sigs", so you would need to specify `q.sig` as `sigs/q.sig` if you are using `utils.runscript(..., in_location=location)`.

Comment (marisalim) [reply tests/test_lca.py:1539]: got it, thanks :)

**Pull request merged**